nextjs-app/components/pages/program-rules/Schedule/StepDetails.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from "react";
+import { IScheduleStep } from "@/types/schedule";
+import { splitArray } from "@/utils/marquee";
```

Comment:
If we want to use this one here, maybe we should put the utils to more general place instead of `marquee`